geodiff/src/geodiff-cli.cpp

```diff
@@ -682,6 +682,25 @@ static int handleCmdVersion( const std::vector<std::string> &args )
   return 0;
 }
 
+static GEODIFF_LoggerLevel getGeodiffLoggerLevel( )
+{
+  int ret = ( int ) LevelWarning;
+#ifdef WIN32
+  char *val = nullptr;
+  size_t sz = 0;
+  if ( _dupenv_s( &val, &sz, "GEODIFF_LOGGER_LEVEL" ) == 0 && val != nullptr )
+  {
+    ret = atoi( val );
+    free( val );
+  }
+#else
+  char *val = getenv( "GEODIFF_LOGGER_LEVEL" );
+  if ( val )
+    ret = atoi( val );
+#endif
```

Comment:
why not use getEnvVar / getEnvVarInt?

Reply:
I think getEnvVarInt is not in public geodiff.h header, so not sure if I can use it?